src/software/geom/util.cpp

```diff
@@ -1381,15 +1385,15 @@ std::vector<Circle> findOpenCircles(Rectangle bounding_box, std::vector<Point> p
         // vector connecting the two points. Return 2 circles that are centered at the
         // points where the splitting vector intercepts the bounding_box. We should also
         // include circles centered at each of the corners.
-        Point connectedVec            = points[1] - points[0];
+        Vector connectedVec           = points[1] - points[0];
         Point halfPoint               = points[0] + (connectedVec * 0.5);
-        Point perpVec                 = Point(connectedVec.y(), -connectedVec.x()).norm();
+        Vector perpVec                = connectedVec.perpendicular();
         std::vector<Point> intersects = lineRectIntersect(
             bounding_box,
             halfPoint +
                 (perpVec * dist(bounding_box.furthestCorner(halfPoint), halfPoint)),
-            halfPoint -
-                (perpVec * dist(bounding_box.furthestCorner(halfPoint), halfPoint)));
+            halfPoint +
+                -((perpVec * dist(bounding_box.furthestCorner(halfPoint), halfPoint))));
```

Comment:
Should this still be `+ -(...`?

Reply:
This was leftover from when I didn't have the new operator. Fixed in latest update